drivers/driver.go

```diff
@@ -42,6 +42,13 @@ type CreateOpts struct {
 	StorageOpt map[string]string
 }
 
+// MountOpts contains optional arguments for Get() methods.
```

Comment:
_Mount_ options for _Get_ methods? The name seems right (external callers call `LayerStore.Mount`, `ProtoDriver.Get` is an internal implementation detail of the driver interface), but for the same reason the documentation should probably focus on `LayerStore.Mount` rather than `Get`.